src/core/middlewares/registryproxy/handler.go

```diff
@@ -110,13 +118,19 @@ func singleJoiningSlash(a, b string) string {
 	return a + b
 }
 
-func handlerPutManifest(res *http.Response) error {
+func handlePutManifest(res *http.Response) error {
+
+	// treat manifest as a blob, it needs to insert blob info.
+	err := handlePutBlob(res)
+	if err != nil {
+		log.Error(err)
+	}
+
 	mfInfo := res.Request.Context().Value(util.MFInfokKey)
 	mf, ok := mfInfo.(*util.MfInfo)
 	if !ok {
 		return errors.New("failed to convert manifest information context into MfInfo")
```

Comment:
Should we return the error or just log it as the image has already been pushed?

Reply:
yes, no error is returned and just log it.